source/multigrid/mg_tools.cc

```diff
@@ -1511,8 +1511,9 @@ namespace MGTools
 
     for (; cell!=endc; ++cell)
       {
+        // do not look at artificial cells
         if (mg_dof_handler.get_tria().locally_owned_subdomain()!=numbers::invalid_subdomain_id
-            && cell->level_subdomain_id()!=mg_dof_handler.get_tria().locally_owned_subdomain())
+            && cell->level_subdomain_id()==numbers::artificial_subdomain_id)
```

Comment:
we should eventually introduce something like `is_level_ghost()` and `is_level_artificial()`.